peps/pep-0746.rst

```diff
@@ -45,62 +45,47 @@ Specification
 =============
 This PEP introduces a protocol that can be used by static and runtime type checkers to validate
 the consistency between ``Annotated`` metadata and a given type.
-Objects that implement this protocol have a method named ``__supports_type__``
-that takes a single positional argument and returns ``bool``::
+Objects that implement this protocol have an attribute called ``__supports_type__``
+that specifies whether the metadata is valid for a given type.
 
     class Int64:
-        def __supports_type__(self, obj: int) -> bool:
-            return isinstance(obj, int)
+        __supports_type__: int
 
-The protocol being introduced would be defined as follows if it were to be defined in code form::
+The attribute may also be marked as a ``ClassVar`` to avoid interaction with dataclasses:
```

Comment:
CI error:
```
docutils.utils.SystemMessage: /home/runner/work/peps/peps/peps/pep-0746.rst:61: (ERROR/3) Unexpected indentation.
```

We need to end with a double colon to indicate the next chunk is code:

```suggestion
The attribute may also be marked as a ``ClassVar`` to avoid interaction with
dataclasses::
```

and remember: "Lines should usually not extend past column 79"

https://peps.python.org/pep-0012/#general

Reply:
Got it thanks. I think a lot of lines go past column 79 in this document, I'd propose we make a followup PR that just reformats to keep the change log clear.